datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -52,3 +54,12 @@ def transformer(sources, **kwargs):
         transformer = column_transformer
 
     return transformer
+
+
+def normalize_datetime(dt):
+    # Prevent naive datetime objects
+    if dt.tzinfo is None:
+        # The stdlib datetime.timezone.utc doesn't work properly on Windows
+        dt = dt.replace(tzinfo=pytz.utc)
```

Comment:
We are assuming that since this is coming from a database, its most likely UTC which makes sense.  

There are occasions if we have a naive timestamp that we should normalize to local timezone instead, maybe an opportunity for a future PR timeutils module.

An implementation I have used in the past which has been quite handy: https://github.com/riverbed/steelscript/blob/af4405c63f378db5466397fd566beb8226edf647/steelscript/common/timeutils.py#L52